### DISCUSSION TITLE
Oink code

### DISCUSSION THREAD

**Discussion by y0d4a** [Security-Onion-Solutions/securityonion#11149] [2.4]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Hi, in latest version, i want to add etpro ruleset, in configuration i see that i need to enter registration code, ruleset and urls.
I am not aware what kind of URL i need to enter (https://rules.emergingthreatspro.com/$oinkcode/suricata-$version/etpro.rules.tar.gz
? )? i got OINK code only...

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Are you doing this through the new 2.4 web interface?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/dc0fa222-64f0-44c0-b0db-a41936af06d9)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/67a87e41-fbc6-490f-a63e-338476dea6b3)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/8c3ef684-ee56-4fbf-8816-f7815109d13a)

**Comment by y0d4a** [comment]
yeap and i successful setup, i just enter ETPRO and key (without url).
on update, i saw that he grab PRO signatures.

Thank you for answer
